core/src/backends/lnd/rest/types.rs

```diff
@@ -253,3 +255,65 @@ impl TryInto<PayInvoiceResult> for SendPaymentSyncResponse {
         Ok(result)
     }
 }
+
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize)]
```

Comment:
```suggestion
#[derive(Default, Debug, Clone, PartialEq, Deserialize)]
```
We don't need the `Serialize` trait here as we only get that type in response to a request.